worker/deps/getopt/getopt.gyp

```diff
@@ -0,0 +1,18 @@
+{
+  'targets': 
+  [
+    {
+      'target_name': 'getopt',
+      'type': 'static_library',
+      'sources':
+      [
+        'getopt.h',
+        'getopt.cpp'
+      ],
+      'direct_dependent_settings':
+      {
+        'include_dirs': ['.']
+      },
+    }
+  ]
+}
```

Comment:
Add line break at the end of all files :)

Reply:
NOTE: You have added all `getopt` dep into `worker/deps/getopt/` folder, however there is no "getopt" entry in `worker/scripts/get-dep.sh`. We prefer to have all deps with their corresponding entry in `get-dep.sh` pointing to a GIT repository and specific branch/version.

Instead of copying `getopt` files manually, is it possible to use some GIT repo instead?

And, if so, since such a repo probably won't have any `getopt.gyp`, that's why I told about creating a main `worker/deps/getopt/` folder as follows:

```
worker/
  deps/
    getopt/
      getopt.gyp  <-- The one you have created (must change paths).
      getopt/     <-- The result of git clone (by scripts/get-dep.sh).
        getopt.c
        getopt.h
```

Reply:
Got. updated.